.github/workflows/lint.yaml

```diff
@@ -0,0 +1,11 @@
+name: docker-lint
+on: [pull_request]
+jobs:
+  # Can look up error with https://github.com/hadolint/hadolint/wiki/DL3029
+  docker-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+      - uses: actions/checkout@v3
+
+      - run: docker run -v --rm -i hadolint/hadolint hadolint --failure-threshold warning - < great_expectations_cloud/agent/Dockerfile
```

Comment:
If you put this into an `invoke` task you can make it part of our existing `lint` matrix.

https://github.com/great-expectations/cloud/blob/fb172df6ebb6b1bca86f1a465aa50959da0ed923/.github/workflows/ci.yaml#L12

Reply:
yep all set